doc/dev_setup.md

```diff
@@ -23,15 +23,15 @@
 
 ## Create a virtual environment
 
-1. From the `AzureDevOpsCli` directory, create a new virtual environment:
+1. From the `vsts-cli` directory, create a new virtual environment:
    ```bash
    virtualenv env
```

Comment:
Add gir checkout for our branch since it is not default branch and default branch has the same named scripts in the script directory.